### PR TITLE
Added Anonymous to crossOrigin prop

### PR DESCRIPTION
**Pull request by pxwee5**
Discussion: https://github.com/szimek/signature_pad/issues/399
Tested and it works for my setup. What do we need to get this through ? @szimek

### CODE REVIEW

File: src/signature_pad.ts [hunk 1]
```diff
@@ -138,6 +138,7 @@ export default class SignaturePad {
         callback(error);
       }
     };
+    image.crossOrigin = 'Anonymous';
```
Comment (UziTech) [inline src/signature_pad.ts:141]: Not sure if it matters but the spec shows it as lowercase. [mdn](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/crossorigin)
```suggestion
    image.crossOrigin = 'anonymous';
```

**Review by UziTech — CHANGES_REQUESTED** [review]

**Review by UziTech — APPROVED** [review]

**Review by szimek — APPROVED** [review]

**Comment by pxwee5** [comment]
Any plans to do a patch release in the 2.x.x branch ?

**Comment by szimek** [comment]
@pxwee5 Yeah, thought it's really hard to say when exactly. Do you have a list of bug fixes that were merged into 3.x branch that you'd like to have in 2.x release? This would help a lot!

**Comment by pxwee5** [comment]
Hey @szimek. I'm not aware of any other bug that was fixed in v3 that needs to be in v2. If there are any that you think should fix v2 we should definitely list them out. 
But in my opinion, the `crossOrigin` fix alone is critical enough to warrant a patch fix for v2.

**Pull request merged**